packages/server/modules/serverinvites/tests/invites.spec.js

```diff
@@ -46,7 +47,7 @@ async function validateInviteExistanceFromEmail(emailParams) {
   // Validate that invite exists
   const token = getInviteTokenFromEmailParams(emailParams)
   expect(token).to.be.ok
-  const invite = await getInviteByToken(token)
+  const invite = await findInviteByToken({ db: knexInstance })(token)
```

Comment:
How are we approaching the creating of a new baseline (prototype/example) for testing that we discussed yesterday? Is that gonna be a separate thing?